aerospike_helpers/cdt_ctx.py

```diff
@@ -77,13 +77,41 @@
     print(result)
     # {'users': 4}
 
+    # Example 3: index_cdt_create() with info command
```

Comment:
This doesn't use index_cdt_create(). It should be called something like "create a CDT secondary index from a base64 encoded _cdt_ctx".